RI.md

```diff
@@ -0,0 +1,842 @@
+| ri Enumerable                | Document for Ruby module Enumerable.                       |
+| ri Arr                       | Document for Ruby class Array (unique initial characters). |
+| ri Nokogiri::HTML4::Document | Document for gem class Nokogiri::HTML4::Document.          |
+| ri Nokogiri                  | Document for gem module Nokogiri.                          |
+<br>
+
+If [option \\--all][4]
+is in effect, documents for the methods in the named class or module
+are included in the display.
+
+### Names for Method Documents
+
+These example `ri` commands cite names for method documents
+(see [details and examples][5]):
+
+| Command                              | Shows                                                                            |
+|--------------------------------------|----------------------------------------------------------------------------------|
+| ri IO::readlines                     | Document for Ruby class method IO::readlines.                                    |
+| ri IO#readlines                      | Document for Ruby instance method IO::readlines.                                 |
+| ri IO.readlines                      | Documents for Ruby instance method IO::readlines and class method IO::readlines. |
+| ri ::readlines                       | Documents for all class methods ::readlines.                                     |
+| ri #readlines                        | Documents for all instance methods #readlines.                                   |
+| ri .readlines, ri readlines          | Documents for class methods ::readlines and instance methods #readlines.         |
+| ri Nokogiri::HTML4::Document::parse  | Document for gem class method Nokogiri::HTML4::Document::parse.                  |
+| ri Nokogiri::HTML4::Document#fragment | Document for gem instance method Nokogiri::HTML4::Document#fragment.            |
+<br>
+
+### Names for Page Documents
+
+These example `ri` commands cite names for page documents
+(see [details and examples][6]):
+
+| Command                        | Shows                                            |
+|--------------------------------|--------------------------------------------------|
+| ri ruby:syntax/assignment.rdoc | Document for Ruby page assignment.               |
+| ri ruby:syntax/assignment      | Same document, if no other syntax/assignment.*.  |
+| ri ruby:assignment             | Same document, if no other */assignment.*.       |
+| ri nokogiri:README.md          | Document for page README.md.                     |
+<br>
+
+### Names for Lists
+
+These example `ri` commands cite names for lists
+(see [details and examples][7]):
+
+| Command       | Shows                   |
+|---------------|-------------------------|
+| ri ruby:      | List of Ruby pages.     |
+| ri nokogiri:  | List of Nokogiri pages. |
+<br>
+
+There are more lists available;
+see [option \\--list][8].
```

Comment:
This look similar to to https://github.com/ruby/rdoc/pull/1100/files#r1618865177 

(by @colby-swandale)

> Is the use of `\` necessary here as it's hard to tell what it's for? Should we perhaps be wrapping these in code blocks?

Reply:
Same as there.  Stet, I think.